### DISCUSSION TITLE
Details List - Control default sort column

### DISCUSSION THREAD

**Discussion by JorgeCFDaniel** [microsoft/powercat-creator-kit#225] [Q&A] [answered]
How to set a default sort column for a details list?

Columns are sortable as set in columns_items. However, when the list is first shown it doesn't seem possible to set the column on wich it should be sorted on.

For instance, in an app for managing accounts receivable it makes sense to sort on the due date.

Regards,
Jorge Daniel

**Comment by denise-msft** [comment]
Thanks for posting your question @JorgeCFDaniel.

Did you try implementing automatic sorting using the configuration described in the documentation? Link to section with sorting: https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/detailslist#sort-events

Also, I'm transferring this to a discussion since this is not a bug or feature request.

**Comment by yaseralhosani** [comment, marked as answer]
Hi @JorgeCFDaniel 

I think this video will help you. 
[Dataverse Delegable Paging in Power Apps](https://www.youtube.com/watch?v=zfyoBLbPvoQ&t=869s) by @scottdurow
